frontend/public/src/components/EnrolledItemCard.js

```diff
@@ -543,10 +541,11 @@ export class EnrolledItemCard extends React.Component<
                     >
                       Email Settings
                     </DropdownItem>
-                    {this.renderEmailSettingsDialog(enrollment)}
                   </span>
                 </DropdownMenu>
               </Dropdown>
+              {this.renderRunUnenrollmentModal(enrollment)}
+              {this.renderEmailSettingsDialog(enrollment)}
```

Comment:
**Quick explanation:** Reactstrap Dropdowns attach event handlers to all children that call `preventDefault` on keydown events, see https://github.com/reactstrap/reactstrap/blob/6085ac0bb340f3e2cdae1aec4e6abec38c37ebad/src/Dropdown.js#L354. Normally, `onClick` handlers will fire on a keydown event bubbling from a button element. But if you call preventDefault on the keydown event, that behavior is prevented.

There's no reason, afaik, that the modal needs to be a child of the dropdown, so moving it outside the dropdown fixes the issue.